spanner/src/add_and_drop_database_role.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/spanner/README.md
+ */
+
+namespace Google\Cloud\Samples\Spanner;
+
+// [START spanner_add_and_drop_database_role]
+use Google\Cloud\Spanner\SpannerClient;
+
+/**
+ * Adds and drops roles to the Singers table in the example database.
+ * Example:
+ * ```
+ * add_and_drop_database_role($instanceId, $databaseId);
+ * ```
+ *
+ * @param string $instanceId The Spanner instance ID.
+ * @param string $databaseId The Spanner database ID.
+ */
+function add_and_drop_database_role($instanceId, $databaseId)
```

Comment:
```suggestion
function add_and_drop_database_role(string $instanceId, string $databaseId): void
```

Reference: https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/spanner/src/pg_create_database.php#L36

Reply:
nit: Probably can remove `and` from the file name and the method.